modules/rosa-sts-creating-cluster.adoc

```diff
@@ -90,15 +103,25 @@ You can configure the following default network IP ranges:
 
 For the CIDR-related `rosa` CLI arguments, see `rosa create cluster --help | grep cidr`. In the interactive mode, you are prompted for the settings.
 ====
-
-* To create a cluster with STS using interactive prompts:
++
+[NOTE]
+====
+The cluster state is `Pending` until the following steps are complete.
+====
++
+* To create a cluster with STS using the defaults:
 +
 [source,terminal]
 ----
-$ rosa create cluster --interactive
+$ rosa create cluster --cluster-name ${name} --sts
 ----
++
+[IMPORTANT]
+====
+To use the `--sts` option to create the cluster, you must use the default `ManagedOpenShift` prefix to create the account-wide roles and policies in the preceding step. The cluster will be created using the account roles and policies that include the default `ManagedOpenShift` prefix. 
```

Comment:
Actually, user doesn't need to use 'ManagedOpenShift' to create account-roles. When user create cluster with --sts it will go into interactive to let user choose the created roles. The created roles with command `rosa create account-roles` will be listed in the interactive mode.

Reply:
@xueli181114 thank you for clarifying this! I was unsure if I could run this in the shared AWS account I am using so I could not test it. Now that you have confirmed that it enters an interactive mode, I will run the command to see the output and update the note according to your guidance.

Reply:
@xueli181114 as discussed, in my testing just now the `--sts` mode automatically picked the only account-wide roles in the AWS account for that cluster version and did not prompt me. I have subsequently installed the 1.1.2 version of `rosa` as per our discussion and tested the cluster create methods again. I've added/updated the example outputs for each option so that it is clear to the reader what will happen when they run the commands.

I have also added a note under the `--sts` option stating:

"If more than one matching set of account-wide roles are available in  your account for a cluster version, an interactive list of options is  provided"